dashboard/test/ui/features/support/cookie_helpers.rb

```diff
@@ -30,6 +30,12 @@ def decrypt_cookie(cookies)
 
     encryptor.decrypt_and_verify(message)
   end
+
+  # It's safer to retrieve a cookie from the list of all cookies
+  # because `#cookie_named` may raise `Selenium::WebDriver::Error::UnknownMethodError` during mobile testing.
+  def get_cookie(key)
+    @browser.manage.all_cookies.find {|cookie| cookie[:name] == key}
+  end
```

Comment:
The error: https://cucumber-logs.s3.amazonaws.com/test/test/iPad_cookie_dcdo_output.html?versionId=Byg7lZiczljR7d7cQ5gpJNVzxOfKT99Y